packages/react-instantsearch-core/src/connectors/connectMenu.js

```diff
@@ -112,21 +112,20 @@ export default createConnector({
     );
 
     // Search For Facet Values is not available with derived helper (used for multi index search)
-    if (searchable && this.context.multiIndexContext) {
-      throw new Error(
-        'react-instantsearch: searching in *List is not available when used inside a' +
-          ' multi index context'
-      );
-    }
+    // @TODO: re-enable this error
```

Comment:
Can we already re-enable it?

Reply:
The error only makes sense once multi-index is enabled, so I did in #2185

Reply:
Yep so it with my review of the PR.